lua/starfall/libs_sh/entities.lua

```diff
@@ -829,6 +829,48 @@ function ents_methods:getBonePosition(bone)
 	return vwrap(pos), awrap(ang)
 end
 
+--- Returns the angle manipulate angle of the bone (local to the entity)
```

Comment:
local to parent bone

Reply:
ive just changed that to (relative to its default angle) as you posted this, would it be better as 'local to parent bone'?

Reply:
You also by mistake changed it in the `getManipulateBonePosition`

Reply:
ah shit that wasnt by mistake i forgot to change the angle to position tho